collection/Emperror; Emperror's New Groove.json

```diff
@@ -7967,31 +7013,15 @@
 			},
 			"entries": [
 				"You endow yourself with endurance and martial prowess fueled by magic. Until the spell ends, you can't cast spells, and you gain the following benefits:",
-				"You gain 100 temporary hit points. If any of these remain when the spell ends, they are lost.",
-				"You have advantage on attack rolls that you make with simple and martial weapons.",
-				"When you hit a target with a weapon attack, that target takes an extra {@damage 4d12} force damage.",
-				"You have proficiency with all armor, shields, simple weapons, and martial weapons.",
-				"You have proficiency in Strength and Constitution saving throws.",
-				"You can attack twice, instead of once, when you take the  Attack action on your turn. You ignore this benefit if you already have a feature, like Extra Attack, that gives you extra attacks.",
-				"You can magically equip any item to which you are attuned as part of casting this spell. Weapons and shields appear in hand, and armor teleports onto your person."
+				"<ul><li>You gain 100 temporary hit points. If any of these remain when the spell ends, they are lost.</li><li>You have advantage on attack rolls that you make with simple and martial weapons.</li><li>When you hit a target with a weapon attack, that target takes an extra {@damage 4d12} force damage.</li><li>You have proficiency with all armor, shields, simple weapons, and martial weapons.</li><li>You have proficiency in Strength and Constitution saving throws.</li><li>You can attack twice, instead of once, when you take the  Attack action on your turn. You ignore this benefit if you already have a feature, like Extra Attack, that gives you extra attacks.</li><li>You can magically equip any item to which you are attuned as part of casting this spell. Weapons and shields appear in hand, and armor teleports onto your person.</li></ul>"
```

Comment:
For lists use the list syntax as given in examples in the renderer demo:
https://5e.tools/renderdemo.html

```json
{
	"type": "list",
	"items": [
		"Item 1",
		"Item 2",
		"Item 3"
	]
}
```